src/vocutouts/workers.py

```diff
@@ -100,11 +100,16 @@ def cutout_range(
     repository = os.environ["CUTOUT_BUTLER_REPOSITORY"]
     collection = os.environ["CUTOUT_BUTLER_COLLECTION"]
     butler = Butler(repository, writeable=True, run=input_collection)
```

Comment:
I'm pretty sure that if a `run` is specified then `writeable` is implied. It is though confusing to set `run` to the "input collection" since from this butler's point of view that's an output collection. If this "input collection" is solely somewhere to store the positions catalog it might be less jarring to call the variable "positions_collection"